db/version_set.cc

```diff
@@ -2112,6 +2112,8 @@ bool Version::MaybeInitializeFileMetaData(FileMetaData* file_meta) {
 }
 
 void VersionStorageInfo::UpdateAccumulatedStats(FileMetaData* file_meta) {
+  TEST_SYNC_POINT("VersionStorageInfo::UpdateAccumulatedStats");
```

Comment:
Nit: maybe use `TEST_SYNC_POINT_CALL_BACK`.

Reply:
`TEST_SYNC_POINT_CALLBACK` is only needed when you want to pass a pointer to the callback, right? At least, that's my understanding based on the header :)

```
  // The argument to the callback is passed through from
  // TEST_SYNC_POINT_CALLBACK(); nullptr if TEST_SYNC_POINT or
  // TEST_IDX_SYNC_POINT was used.
  void SetCallBack(const std::string& point,
                   const std::function<void(void*)>& callback);
```

Reply:
That's correct. In this situation, I usually pass a `nullptr` and use `TEST_SYNC_POINT_CALL_BACK`. In this PR, this is not necessary.